src/Anonymization/Anonymizer/Core/Md5Anonymizer.php

```diff
@@ -11,7 +11,11 @@
 #[AsAnonymizer(
     name: 'md5',
     pack: 'core',
-    description: 'Anonymize a column by hashing its value.'
+    description: <<<TXT
+    Anonymize a column by hashing its value.
+    Options are 'use_salt' (default true).
+    Using a salt prevents reverse hash of values using rainbow tables.
```

Comment:
I think "... prevents reversing (the) hash of ..." or "... prevents reverse hashing of ..." would be a more correct wording.